pycloudlib/ec2/cloud.py

```diff
@@ -215,7 +215,7 @@ def list_keys(self):
             keypair_names.append(keypair["KeyName"])
         return keypair_names
 
-    def snapshot(self, instance, clean=True):
+    def snapshot(self, instance, clean=True, wait=True):
```

Comment:
Good thought in expediting pycloud lib run, this takes quite a while on ec2 sometimes.